lib/delivery_estimator.rb

```diff
@@ -0,0 +1,5 @@
+class DeliveryEstimator
+  def eta
+    Time.now + 60 * 60
+  end
+end
```

Comment:
Over-engineered: Here you have a class that has a single responsibility, but it is a vacuous responsibility.